docs/design/mono/diagnostics-tracing.md

```diff
@@ -122,7 +118,54 @@ Prerequisites when running below diagnostic scenarios:
 * If anything EventSource related is used, make sure EventSourceSupport is enabled when building application (or ILLinker can remove needed EventSource classes). If just running tracing, collecting native EventPipe events emitted by SampleProfiler, DotNetRuntime or MonoProfiler providers, EventSourceSupport can be enabled or disabled.
 * Install needed diagnostic tooling on development machine.
 
-### Application running diagnostics and connecting over loopback interface
+### Application running diagnostics using default .NET 8 configuration
+
+Starting in .NET 8, enhancements to `dotnet-dsrouter` supporting most of below described scenarios, using a smaller subset of available configurations using loopback interface on emulators/simulators and physical devices attached over usb. `dotnet-dsrouter` outputs detailed information on how to launch runtime as well as connect to the instance using diagnostic tooling.
+
+Starting `dotnet-dsrouter` to trace an application running on iOS simulator could be done using the `ios-sim` profile:
+
+```
+dotnet-dsrouter ios-sim -i
+WARNING: dotnet-dsrouter is a development tool not intended for production environments.
```

Comment:
To help with telling commands apart from text, other documentation often uses `$` as a prefix for commands.
````suggestion
```sh
$ dotnet-dsrouter ios-sim -i
WARNING: dotnet-dsrouter is a development tool not intended for production environments.
````